File: 2.0/python/texts/getTextBroadcastStats.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.texts.getTextBroadcastStats(
    id=11646003,
    # filter by time interval
    begin=1473781817000,
    # filter by time interval
    end=1473781817000,
    # return only specific fields
    fields='totalOutboundCount,billedAmount,sentCount'
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)

File: 2.0/php/texts/getTextBroadcastStats.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->getTextBroadcastStats();
        $request->getOperationConfig()->setPathParameters(array("id" => 11646003));
        $request->getOperationConfig()->setQueryParameters(array("fields" => "totalOutboundCount,billedAmount,sentCount",
                                                                 "begin" => 1473781817000,
                                                                 "end" => 1473781817000));
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();
